docs/source/getting-started/install-daml-finance.rst

```diff
@@ -0,0 +1,37 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Getting started : Installing Daml Finance
```

Comment:
I feel like the title "Installing Daml Finance" might be a bit misleading, as it's really just downloading the code (that is relevant for the subsequent tutorials). Besides, maybe it would be easier if the code we ship as part of the assistant template would cover all three "getting started" tutorials.

But happy for this to be tackled in subsequent PRs.